flows/actions/open_ticket.go

```diff
@@ -75,9 +74,17 @@ func (a *OpenTicketAction) Execute(ctx context.Context, run flows.Run, step flow
 
 	ticket := a.open(run, topic, assignee, evaluatedNote, logModifier, logEvent)
 	if ticket != nil {
-		a.saveResult(run, step, a.ResultName, string(ticket.UUID()), CategorySuccess, "", "", nil, logEvent)
+		run.Locals().Set("_new_ticket", string(ticket.UUID()))
```

Comment:
@ericnewcomer @norkans7 still not sure what the naming scheme should be for locals that users don't define.. maybe not too worried here because I doubt anyone is using this value